content/en/blog/falco-0-37-0/index.md

```diff
@@ -0,0 +1,293 @@
+
+## Breaking changes
+
+This is a list of breaking changes introduced in Falco 0.37.0
+
+* The [Rate-limiter mechanism](https://github.com/falcosecurity/falco/pull/2841) was removed as it is no longer used.
+
+* [`--userspace CLI option`](https://github.com/falcosecurity/falco/pull/2839) was removed as it’s no longer used.
+
+* The `falco-driver-loader` script is removed and [embedded into falcoctl](https://github.com/falcosecurity/falco/pull/2905)
+
+* The Helm chart 4.0.0 contains several modifications to work with the new k8s metadata collector. Please read its [breaking change](https://github.com/falcosecurity/charts/blob/master/charts/falco/BREAKING-CHANGES.md#400) file for more information.
+
+* The new `falcoctl driver` implementation will drop:
+    * [`--source-only`](https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader#L685)
+    * [`BPF_USE_LOCAL_KERNEL_SOURCES`](https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader#L559)
+    * [`DRIVER_CURL_OPTIONS`](https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader#L692)
+    * The [`FALCO_BPF_PROBE`](https://github.com/falcosecurity/falco/blob/1b62b5ccd1c64cd972ef0252262075cbf42a130c/scripts/falco-driver-loader#L738C9-L738) environment variable won't be used by the new `falcoctl driver` loader as it is already deprecated and scheduled to be removed in the next major version. 
+
+* Various environment variables have been replaced as part of the new `falcoctl driver` feature:
+    * [`DRIVERS_REPO`](https://github.com/falcosecurity/falco/blob/1b62b5ccd1c64cd972ef0252262075cbf42a130c/scripts/falco-driver-loader#L690) has been replaced by `FALCOCTL_DRIVER_NAME`  or the `--name` command line argument.
+    * [`DRIVERS_NAME`](https://github.com/falcosecurity/falco/blob/1b62b5ccd1c64cd972ef0252262075cbf42a130c/scripts/falco-driver-loader#L689) has been replaced by `FALCOCTL_DRIVER_REPOS` or the `--repo` command line argument.
+    * [`DRIVER_KERNEL_RELEASE`](https://github.com/falcosecurity/falco/blob/1b62b5ccd1c64cd972ef0252262075cbf42a130c/scripts/falco-driver-loader#L692) has been replaced by `--kernelrelease` command line argument.
+    * [`DRIVER_KERNEL_VERSION`](https://github.com/falcosecurity/falco/blob/1b62b5ccd1c64cd972ef0252262075cbf42a130c/scripts/falco-driver-loader#L693) has been replaced by `--kernelversion` command line argument. 
+    * [`DRIVER_INSECURE_DOWNLOAD`](https://github.com/falcosecurity/falco/blob/1b62b5ccd1c64cd972ef0252262075cbf42a130c/scripts/falco-driver-loader#L691) has been replaced by `--http-insecure` command line argument.
+
+* [Remove -K/-k options from Falco in favor of the new k8s plugin](https://github.com/falcosecurity/falco/pull/2914)
+
+* [Dropped plugins shipped with Falco](https://github.com/falcosecurity/falco/pull/2997) since plugins will now be managed by `falcoctl`
```

Comment:
```suggestion
* [Dropped plugins shipped with Falco](https://github.com/falcosecurity/falco/pull/2997) since plugins will now be managed by `falcoctl`. If you want to use a plugin like `k8saudit` be sure to install it at init time with `falcoctl`
```